test/ng/parseSpec.js

```diff
@@ -3900,6 +3900,34 @@ describe('parser', function() {
           expect(isFunction(s.toString)).toBe(true);
           expect(l.toString).toBe(1);
         }));
+
+        it('should assign a property to locals properties with null or undefined value', inject(function($parse) {
```

Comment:
The description is misleading. There is no locals object in the test. Use scope instead.